Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackFormViewModel.swift

```diff
@@ -336,19 +336,22 @@ extension SentryUserFeedbackFormViewModel {
         [fullNameTextField, emailTextField].forEach {
             $0.font = config.theme.font
             $0.adjustsFontForContentSizeCategory = true
+            $0.textColor = config.theme.inputForeground
```

Comment:
i guess i'd missed setting the configured text colors for the inputs. this and the messageTextView below